test/upload_test.go

```diff
@@ -70,6 +70,31 @@ func TestUpload_UploadsByUser(t *testing.T) {
 		t.Fatalf("Expected storage used of %d (%d MiB), got %d (%d MiB).",
 			storageUsed, storageUsed/skynet.MiB, stats.StorageUsed, stats.StorageUsed/skynet.MiB)
 	}
+	// Delete the last upload. Expect zero uploads and zero storage used after.
+	unpinned, err := db.UnpinUploads(ctx, *sl, *u)
```

Comment:
What happens if the user uploads the same skylink again after having unpinned it? Should add a test for that

Reply:
Test added.